src/store/application/ui/theme/sagas.ts

```diff
@@ -38,12 +39,15 @@ function* watchNativeTheme() {
 
 export function* watchSetTheme() {
   yield takeEvery(SET_THEME, function* (action: ThemeActions) {
-    setTheme(action.theme)
+    const { theme } = action
+    setTheme(theme)
+    const message = new ThemeChangeMessage(theme)
```

Comment:
i dont think this would break if not native, but maybe check if NATIVE_MOBILE first?

Reply:
👍